config/lava/iec-security/iec-security.jinja2

```diff
@@ -0,0 +1,13 @@
+- test:
+    name: {{ plan }}
+    description: "cyber security standard tests such as IEC-62443-4-2"
+    timeout:
+      minutes: 20
+    definitions:
+    - repository: https://github.com/kernelci/iec-security.git
+      from: git
+      branch: master
```

Comment:
The `kernelci.org` branch is updated during each production update:
```suggestion
      branch: kernelci.org
```

Then there is a patch used on staging to pick the `staging.kernelci.org` branch, but that will get updated once this PR is merged.